Makefile

```diff
@@ -79,6 +79,6 @@ test-workspace:
 		-quiet
 
 format:
-	swift format --in-place --recursive .
+	swift format --in-place --recursive ./Package.swift ./Sources ./Tests
```

Comment:
I thought we made this change earlier. I had some locked artifacts in `.build` that it was tripping on again.